compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/input/TextInputTests.kt

```diff
@@ -49,13 +47,10 @@ import org.w3c.dom.events.CompositionEventInit
 import org.w3c.dom.events.Event
 
 private class InputInteractor(
-    val composeChannel: Channel<String> = Channel<String>(
-        1, onBufferOverflow = BufferOverflow.DROP_OLDEST
-    ),
+    val textFieldValues: MutableList<MutableState<TextFieldValue>> = mutableListOf(),
```

Comment:
Will there ever be a need to specify the list of `TextFieldValue` states and focus requesters here?
You could replace both arguments with an `textFieldCount: Int` argument.